projects/demo/src/modules/components/combo-box/examples/10/index.html

```diff
@@ -0,0 +1,31 @@
+<tui-textfield tuiChevron>
+    <input
+        placeholder="Country"
+        tuiComboBox
+        [(ngModel)]="value"
+    />
+
+    <ng-container *tuiTextfieldDropdown>
+        <!-- 48 = 44 height + 4 margin -->
+        <!-- 8 = data list padding -->
```

Comment:
something is wrong with the example
seems like it has an extra scrollbar and it gets stuck on ios

 https://taiga-previews-demo--pr10980-new-combobox-demo-gis3ii6b.web.app/components/combo-box#virtual-scroll

Reply:
It will be fixed in next PR

Reply:
> seems like it has an extra scrollbar 

Fixed by 
* https://github.com/taiga-family/taiga-ui/pull/11066